subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/api/constants.go

```diff
@@ -97,4 +101,6 @@ const (
 	InspectPath = VersionPath + "/inspect"
 	// Inspect current allocated AffinityGroup(s)
 	AffinityGroupsPath = InspectPath + "/affinitygroups/"
+	// Inspect current cluster status
+	ClusterStatusPath = InspectPath + "/clusterstatus"
 )
```

Comment:
also add sub resources of it, such as:

ClusterStatusPath = InspectPath + "/clusterstatus/"
PCStatusPath = ClusterStatusPath + "physicalcluster"
VCsStatusPath = ClusterStatusPath + "virtualclusters/"

And can also serve:
One specificVC Status:
VCsStatusPath + "{VC NAME}"

This is like AffinityGroups, check its code.

Reply:
Haha, this is just an example. Can we use long name such as physicalcluster instead of PC in API?
And this also align others router path.

---
In reply to: [388111150](https://github.com/microsoft/pai/pull/4153#discussion_r388111150) [](ancestors = 388111150)